GUI/Types/Renderer/PhysSceneNode.cs

```diff
@@ -93,11 +93,26 @@ public PhysSceneNode(Scene scene, PhysAggregateData phys)
                     //m_vCentroid
                     //m_flMaxAngularRadius
                     //m_Vertices
-                    var vertices = hull.GetArray("m_Vertices");
+                    IEnumerable<Vector3> vertices = null;
+                    if (hull is NTROStruct)
+                    {
+                        //NTRO has vertices as array of structs
+                        var verticesArr = hull.GetArray("m_Vertices");
+                        vertices = verticesArr.Select(v => v.ToVector3());
+                    }
+                    else
+                    {
+                        //KV3 has vertices as blob
+                        var verticesBlob = hull.GetArray<byte>("m_Vertices");
+                        vertices = Enumerable.Range(0, verticesBlob.Length / 12)
+                            .Select(i => new Vector3(BitConverter.ToSingle(verticesBlob, i * 12),
+                                BitConverter.ToSingle(verticesBlob, (i * 12) + 4),
+                                BitConverter.ToSingle(verticesBlob, (i * 12) + 8)));
+                    }
                     var vertOffset = verts.Count / 7;
                     foreach (var v in vertices)
```

Comment:
```suggestion
                    foreach (var vec in vertices)
```

unnecessary var assignment

Reply:
Nope. I tried doing that.
> CS1656 vec is a foreach iteration variable